gems/sorbet/test/snapshot/run_one_bazel.sh

```diff
@@ -93,6 +93,17 @@ srb="${repo_root}/gems/sorbet/bin/srb"
 info "├─ sorbet:           $SRB_SORBET_EXE"
 info "├─ sorbet --version: $("$SRB_SORBET_EXE" --version)"
 
+SORBET_TYPED_REV="$(rlocation com_stripe_ruby_typer/gems/sorbet/test/snapshot/sorbet-typed.rev)"
+SRB_SORBET_TYPED_REVISION="$(<"$SORBET_TYPED_REV")"
+export SRB_SORBET_TYPED_REVISION
+
+if [ -z "$SRB_SORBET_TYPED_REVISION" ]; then
```

Comment:
```suggestion
if [ "$SRB_SORBET_TYPED_REVISION" = "" ]; then
```

I don't have a linter for this yet, but we use `= ""` instead of `-z` / `-n` because I think it's too easy to mistake `-n` to mean "not" instead of "non-zero". When using strings as booleans this usually leads to confusion:

```bash
if [ -n "$X" ]

# "if not X" vs "if non-zero X", i.e.
# "if X is false" vs "if X is true"
```

I looked at building this into shellcheck actually and then never got back to it.

Reply:
Makes sense, thanks!